src/shared_modules/utils/socketWrapper.hpp

```diff
@@ -284,6 +284,73 @@ class SizeHeaderProtocol final
     };
 };
 
+/**
+ * @brief This class is used for the following format:
+ *          - M bytes: body. Mandatory data to send.
+ */
+class NoHeaderProtocol final
```

Comment:
Since there are already 3 protocol classes.
Shouldn't be better to create an interface class with the required methods and then derivate the 3 protocol classes from it?

Reply:
I think is a good idea, but I'm not sure if this should be included as the scope of this issue @Dwordcito.